docs/openhabian-amanda.md

```diff
@@ -564,13 +567,81 @@ You could also move that temporary recovered image file to your Windows PC that
+
+```
+head -14 00002.openhab._etc_openhab2.0
+AMANDA: SPLIT_FILE 20200414010002 openhab /etc/openhab2  part 1/-1  lev 0 comp N program /bin/tar
+DLE=<<ENDDLE
+<dle>
+  <program>GNUTAR</program>
+  <disk>/etc/openhab2</disk>
+  <level>0</level>
+  <auth>BSDTCP</auth>
+  <record>YES</record>
+  <index>YES</index>
+  <datapath>AMANDA</datapath>
+</dle>
+ENDDLE
+To restore, position tape at start of file and run:
+	dd if=<tape> bs=32k skip=1 | /bin/tar -xpGf - ... 
+  ```
+
+```
+dd if=00002.openhab._etc_openhab2.0 bs=32k skip=1 | tar tvf -
+drwxrwxr-x openhab/openhab 106 2020-03-15 19:17 ./
+drwxrwxr-x openhabian/openhabian 16 2020-02-09 21:16 ./.vscode/
+drwxrwxr-x openhab/openhab       25 2020-01-27 21:53 ./html/
+drwxrwxr-x openhab/openhab       10 2020-01-27 21:52 ./icons/
+drwxrwxr-x openhab/openhab      884 2020-04-13 20:30 ./icons/classic/
+drwxrwxr-x openhab/openhab       25 2020-01-28 18:03 ./items/
+drwxrwxr-x openhab/openhab       13 2020-01-27 21:53 ./persistence/
+drwxrwxr-x openhab/openhab       25 2020-01-28 18:04 ./rules/
+drwxrwxr-x openhab/openhab       26 2020-01-28 18:04 ./scripts/
+drwxrwxr-x openhab/openhab       53 2020-01-27 21:53 ./services/
+drwxrwxr-x openhab/openhab       27 2020-01-28 18:05 ./sitemaps/
+drwxrwxr-x openhab/openhab       28 2020-01-27 21:52 ./sounds/
+drwxrwxr-x openhab/openhab       26 2020-01-28 18:05 ./things/
+drwxrwxr-x openhab/openhab       40 2020-01-28 18:05 ./transform/
+...
+-rwxrwxrwx openhabian/openhabian 13063 2020-04-08 12:05 ./sitemaps/demo.sitemap
+-rw-rw-r-- openhab/openhab         241 2020-01-13 08:53 ./sitemaps/readme.txt
+-rw-rw-r-- openhab/openhab        8776 2020-01-13 08:53 ./sounds/barking.mp3
+-rw-rw-r-- openhab/openhab      102399 2020-01-13 08:53 ./sounds/doorbell.mp3
+```
+
+```
+dd if=00002.openhab._etc_openhab2.0 bs=32k skip=1 | tar xvf -
+./
+./.vscode/
+./html/
+./icons/
+...
+./transform/de.map
+./transform/en.map
+./transform/readme.txt
+```
+
+You can then put the extracted files back in the correct location on your openhabian box. You do not even have to put back the amanda_data files since amanda will just build new indexes based on the new daily backups, just like it did the first time amanda was installed. It is highly recommended that you make an image backup of the openhabian box SD card as soon as it is fully back in operation. You may have had to reinstall SNAPSHOT.jar binding files, reconfigure exim4 to receive the amanda backup emails and other settings or files not backed up by default by amanda.
```

Comment:
image backup -> backup copy
To do so, include /dev/mmcblk0 in the disklist on your next backup run and restore that to a new SD card (see restore section).

openhabian -> openHABian

amanda -> Amanda

Reply:
I tested the dd to NAS on my setup and it took 3261 sec to do the backup of my 32 GB SD card. Have now added it the monthly cron job as in the updated docs